src/content/docs/es/guides/rss.mdx

```diff
@@ -221,6 +222,20 @@ rss({
 Si prefieres no crear tu propia hoja de estilos, puedes utilizar una hoja de estilos predefinida como la [hoja de estilos predeterminada de Pretty Feed v3](https://github.com/genmon/aboutfeeds/blob/main/tools/pretty-feed-v3.xsl). Descarga la hoja de estilos desde GitHub y guárdala en el directorio `public/` de tu proyecto.
 :::
 
+## Detección automática de fuentes RSS
+
+[RSS autodiscovery](https://www.rssboard.org/rss-autodiscovery) permite a los navegadores y otros programas encontrar automáticamente el canal RSS de un sitio a partir de la URL principal.
+
+Para habilitarlo, agregue una etiqueta `<link>` con los siguientes atributos al elemento `head` de su sitio.
+
+```html
+<link
+    rel="alternate"
+    type="application/rss+xml"
+    title="Título de su sitio web"
+    href={`${Astro.site}rss.xml`}
+/>
+
```

Comment:
For some reason while applying that last suggestion it removed this. Can you add it? 😅
```suggestion

Con esta etiqueta, los lectores de tu blog pueden introducir la URL base de tu sitio en su lector RSS para suscribirse a tus entradas sin necesidad de la URL específica de tu canal RSS.
```